src/blas/backends/portblas/portblas_common.hpp

```diff
@@ -64,6 +68,10 @@ DEF_PORTBLAS_TYPE(oneapi::mkl::transpose, char)
 DEF_PORTBLAS_TYPE(oneapi::mkl::uplo, char)
 DEF_PORTBLAS_TYPE(oneapi::mkl::side, char)
 DEF_PORTBLAS_TYPE(oneapi::mkl::diag, char)
+DEF_PORTBLAS_TYPE(std::complex<float>, sycl_complex_t<float>)
+DEF_PORTBLAS_TYPE(std::complex<double>, sycl_complex_t<double>)
+DEF_PORTBLAS_TYPE(sycl_complex_t<float>, sycl_complex_t<float>)
```

Comment:
This line and the line below it do not look correct to me. Are you sure this is what you want?